third_party/ienn/src/ie_compilation.cpp

```diff
@@ -195,12 +195,13 @@ int32_t Compilation::Compile() {
 
     OutputsDataMap output_info(network_->getOutputsInfo());
     for (auto itr : output_info) {
-      itr.second->setPrecision(Precision::FP32);
+      SizeVector dims = itr.second->getTensorDesc().getDims();
```

Comment:
Why remove this line and add the unused code?

Reply:
@fujunwei I remove the `itr.second->setPrecision(Precision::FP32)` since the output of argmax is int32. I remove the output process code `int32 to float` in `ie_execution.cpp`. The unused code has been removed.